composer.json

```diff
@@ -25,6 +26,8 @@
         "symfony/test-pack": "*",
         "symfony/translation": "*",
         "symfony/twig-pack": "*",
+        "symfony/stimulus-bundle": "*",
```

Comment:
Seems like it's not in alphabetical order ;)

Reply:
> /me checks notes

Hmm, I think you're right :). Thanks - fixed now